app/helpers/proctoring/tokens_helper.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+require 'jwt'
+require 'securerandom'
+
+module Proctoring
+  module TokensHelper
+    def generate_authentication_token(params)
+      generate_app_token(
+        params[:room_id],
+        params[:user_id],
+        params[:role],
+        Proctoring.hundred_ms_app_access_key,
+        Proctoring.hundred_ms_app_secret
```

Comment:
why a new method inside method? there is nothing else inside this other than calling a new method?